src/services/EnsService.ts

```diff
@@ -23,68 +31,188 @@ function getEnsAddress (networkId: string) {
   }
 }
 
+function getRegistrarAddress (networkId: string): string {
+  // .bnb bsc testnet
+  if ([97].includes(parseInt(networkId))) {
+    return '0x888A2BA9787381000Cd93CA4bd23bB113f03C5Af'
+  } 
+  // ens
+  else if ([1, 3, 4, 5].includes(parseInt(networkId))) {
+    return '0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85'
+  }
+  // .bnb bsc mainnet
+  else if ([56].includes(parseInt(networkId))) {
+    return '0xE3b1D32e43Ce8d658368e2CBFF95D57Ef39Be8a6'
+  }
+}
+
+function getTextRecordKeys (): string[] {
+  return [
+    'email',
+    'url',
+    'avatar',
+    'com.discord',
+    'com.github',
+    'com.twitter',
+    'org.telegram',
+  ]
+}
+
+function getAddrRecordKeys (): string[] {
+  return ['ETH', 'BTC', 'LTC', 'DOGE']
+}
+
+function coinTypeOrcoinId2Symbol (coinType: string): string {
```

Comment:
What is `coinId`? There seems to be no such thing.
Even if such a thing exists, the function's name should be `coinTypeOrCoinId2Symbol`. Pay attention to the Uppercase